package.json

```diff
@@ -2,18 +2,16 @@
   "name": "svelte",
   "version": "2.15.1",
   "description": "The magical disappearing UI framework",
-  "main": "compiler/svelte.js",
+  "main": "index.js",
```

Comment:
Should `index.js` (the lifecycle etc hooks functions) be `pkg.module` rather than `pkg.main`? They're not in commonjs, nor do I think they should be.

That would leave the question of 'well what should `pkg.main` be?' I don't know. Maybe nothing?

Reply:
Good question. Maybe nothing? Or maybe we capitulate to Node and have

```js
"main": "index.js",
"module": "index.mjs"
```

Reply:
How about `pkg.main` points to a file that just throws a big old exception and lets the user know what and how they should be importing instead? (Hopefully there aren't any current bundlers that will use `pkg.main` when `pkg.module` is available.) The message could say 'lifecycle stuff lives here, the compiler lives here, animation stuff lives here, observable stuff lives here' etc

Reply:
I've adopted putting `"main": "index"` (without the extension) — as far as I know, according to the current Node modules draft, it will choose the appropriate extension depending on whether it is imported or required.